docs/topics/install-olympia/installation.rst

```diff
@@ -198,10 +226,13 @@ Each incremental change we add to the database is done with a versioned SQL
 (and sometimes Python) file. To keep your local DB fresh and up to date, run
 migrations like this::
 
-    schematic migrations
+    $ schematic migrations/
 
-More info on schematic: https://github.com/mozilla/schematic
+If you want to start from scratch and recreate the database, you can just run
```

Comment:
Maybe you don't need the previous identical sentence above, if you have it here?